model/project_ref.go

```diff
@@ -731,79 +730,54 @@ func FindOneProjectRefByRepoAndBranchWithPRTesting(owner, repo, branch string) (
 		return nil, errors.Wrapf(err, "Could not fetch project ref for repo '%s/%s' with branch '%s'",
 			owner, repo, branch)
 	}
-	l := len(projectRefs)
-	target := 0
-	if l > 1 {
-		count := 0
-		for i := range projectRefs {
-			if projectRefs[i].IsPRTestingEnabled() {
-				target = i
-				count += 1
-			}
-		}
-
-		if count > 1 {
-			return nil, errors.Errorf("attempt to fetch project ref for "+
-				"'%s/%s' on branch '%s' found %d project refs, when 1 was expected",
-				owner, repo, branch, count)
+	for _, p := range projectRefs {
+		if p.IsPRTestingEnabled() {
+			p.checkDefaultLogger()
```

Comment:
Since you removed the error for multiple matching projects, could you delete that part of the comment above?